virttest/libvirt_xml/devices/disk.py

```diff
@@ -231,6 +231,22 @@ def new_backingstore(self, **dargs):
             setattr(new_one, key, value)
         return new_one
 
+    def get_backingstore_list(self):
+        """
+        Usage: get source file attribute from backingStore
+            test_disk = Disk()
+            backingstore_list = []
+            for elem in test_disk.xmltreefile.getiterator('backingStore'):
+                backingstore_list.append(elem)
```

Comment:
please update them as you should using "get_backingstore_list" for your usage

Reply:
@kylazhang 
thanks, now updated